people.json

```
@@ -798,5 +798,14 @@
       "GitHub": "https://github.com/MacDue"
     },
     "contributor": true
+  },
+  {
+    "nick": "Filiph Siitam Sandström",
+    "coordinates": [58.699,  12.453],
```

```suggestion
    "coordinates": [58.699, 12.453],
```

to make the linter happy